cdist/conf/explorer/cpu_sockets

```diff
@@ -25,14 +25,14 @@
 os=$("$__explorer/os")
 case "$os" in
     "macosx")
-        system_profiler SPHardwareDataType | grep "Number of Processors" | awk -F': ' '{print $2}'
+        system_profiler SPHardwareDataType | grep "Number of Processors" | awk -F ': ' '/Number of Processors/ {print $2}'
     ;;
 
     *)
     if [ -r /proc/cpuinfo ]; then
-        sockets="$(grep "physical id" /proc/cpuinfo | sort | uniq | wc -l)"
+        sockets=$(grep "physical id" /proc/cpuinfo | sort -u | wc -l)
```

Comment:
`sort -u` is not defined by POSIX
http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sort.html

Reply:
@uqam-fob Hm... But there it is -u option. Am I getting something wrong?

Reply:
Sorry for the confusion! You're absolutely right. I did not notice that `u` in the corner.